Server/index.js

```diff
@@ -24,7 +24,7 @@ app.get("/products", (req, res) => {
 
 // Headers required: Authorization token
 // Patch request. Get specfic product using product id.
```

Comment:
We should also update the comments to Get for each of these so it isn't confusing.